inst/sql/sql_server/createTargetOutcomeCombinations.sql

```diff
@@ -159,8 +58,8 @@ o.cohort_start_date as outcome_start_date,
 o.cohort_end_date as outcome_end_date,
 t.cohort_definition_id as target_cohort_id,
 o.cohort_definition_id as outcome_cohort_id
-into #target_with_outcome
-from #targets_agg t inner join #outcomes_agg o
+into #target_outcome_tar
+from #targets_inclusions t inner join #outcomes_washout o
 on t.subject_id = o.subject_id
 where
 -- outcome starts before TAR end
```

Comment:
I think it's safe to say that the outcome is a case when it starts within tar start and start end. Where it ends doesn't matter.

Reply:
I was commenting out old code - I sometimes comment while making changes in case I want to go back.  We can remove these two lines of comments.